src/styles/blocks/catalog.scss

```diff
@@ -17,9 +18,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
```

Comment:
Please use variable for transition time

It's better to aslo specify transition property (color, transform, etc.) directly, please add an transition-property rule, or just use shorthand like transition: transform $transition-time